project1/codesubmission/clockwise/roomba-roam.cc

```diff
@@ -0,0 +1,60 @@
+// mathematical constants to help with rotating the robot in radians e.g. M_PI
+#define _USE_MATH_DEFINES
+
+#include <cmath>
+#include "Robot.h"
+
+// forward declarations
+double feetToMeter(double);
+void drawSquare(Robot&, double, bool);
+
+int main(int argc, char *argv[])
+{
+  // Initialize our robot with isSimulation set to true
+  Robot robot;
```

Comment:
I'm assuming he also wants us to give him the code as it was for when we ran the real robot. Sorry I didn't inform you about this sooner! Simply comment out this line